readthedocs/analytics/proxied_api.py

```diff
@@ -25,6 +26,8 @@ class BaseAnalyticsView(APIView):
     - absolute_uri: Full path with domain.
     """
 
+    # Never cache this view, we always want to hit this endpoint.
```

Comment:
```suggestion
    # We always want to hit our analytics endpoint,
    # so we capture 100% of known views/interactions. 
```

Comment the `why`, not the `what` 💯

Reply:
Why we add the `CDNCacheControlMixin` for requests we are sure we don't want to cache? Is there any benefit on returning `Cache-Control: private` instead of not returning this header at all?

Reply:
@humitos we cache everything by default on .org, so views need to opt out of this behavior.